src/elements/content-sidebar/flowTypes.js

```diff
@@ -11,8 +12,14 @@ type NavigateOptions = {
     isToggle?: boolean,
 };
 
+type AdditionalSidebarTabFtuxData = {
```

Comment:
Is this unique to this particular component or is it a more generic type?

Reply:
Specific to this component, but I guess there's no reason why another component doing something similar couldn't use it. The data is passed into the component from the consumer of the sidebar, so anyone else that wants to use it would need to make sure that they're using the same format to pass data in